test/test_model.py

```diff
@@ -52,7 +88,20 @@ def test_three_layer(input, compare):
     """
     print( input.shape )
     #Construct the model
-    layers = [10, 15]
-    model = Model(1,1, layers)
+    config.hidden_size = [10]
+    model = Model(config)
     #Pass input through the model
-    tmp = model(input)
\ No newline at end of file
+    tmp = model(input)
+    
+@pytest.mark.usefixtures("data_dims")
+def test_data_load(data_dims: Tuple[int, int]):
+    """_summary_
+
+    Args:
+        data_dims (_type_): _description_
+    """
+    #Load the data
+    runoff_mean, precip_mean = load_data()
+    #Check the shapes
+    assert runoff_mean.shape == data_dims, "Runoff mean shape is incorrect"
+    assert precip_mean.shape == data_dims, "Precip mean shape is incorrect"
```

Comment:
Add a new line for posix.

Reply:
I added a check for the data type using `isinstance`. Is that what you meant by posix?

Reply:
No, quite literally just a new line at the end of the file...[for quick explanation](https://www.baeldung.com/linux/files-end-with-newlines#:~:text=Newlines%20at%20the%20End%20of,defined%20as%20consisting%20of%20lines.)

Reply:
done 👍 Thanks for the link/reading